surface/model_openapiv3.go

```diff
@@ -189,11 +163,11 @@ func (b *OpenAPI3Builder) buildFromNamedPath(name string, pathItem *openapiv3.Pa
 	}
 }
 
-// Builds the "Parameters" and "Responses" types for an operation, adds them to the model, and returns the names of the types.
+// Builds the "Request" and "Responses" types for an operation, adds them to the model, and returns the names of the types.
 // If no such Type is added to the model an empty string is returned.
 func (b *OpenAPI3Builder) buildFromNamedOperation(name string, operation *openapiv3.Operation) (parametersTypeName string, responseTypeName string) {
 	// At first, we build the operations input parameters. This includes parameters (like PATH or QUERY parameters) and a request body
-	operationParameters := makeType(name + "Parameters")
+	operationParameters := makeType(name + "Request")
```

Comment:
Are we sure we want to change the name here? "Parameters" is the OpenAPI schema property name, so I'd say it has some contextual value. I like "Request" because it lines up with protobuf style, but I'm not sure we want to do that here. 

Also, I don't quite remember, is this name used in a plugin/user-facing API? Meaning, will a downstream consumer (a plugin, an end-user of gnostic, etc) be depending on the `Name` being `FooParameters` (i.e. they are generating code with that type name)? If so, changing the constructed name could cause problem for downstream tools/users.

Reply:
Yes, I agree with you Noah. We should change the suffix in gnostic-grpc.

I think gnostic-go-generator would have generated something like:
```
type FooRequest {
   ...
}
```
Instead of:
```
type FooParameters {
   ...
}
```

Reply:
Ok, yeah good call to revert the change then. gnostic-grpc seems like the right spot for such a change 😄